src/request_handler.rb

```diff
@@ -0,0 +1,69 @@
+require 'yaml'
+
+class RequestHandler
+  attr_reader :message, :bot, :receiver
+
+  def initialize(receiver)
+    @receiver = receiver
+    @message = receiver.message
+    @bot = receiver.bot
+
+    @answers = YAML.load_file(
+      File.join(File.dirname(__FILE__), 'data', 'messages.yml')
```

Comment:
It loads keys as `'strings'`. Then you fetching them as `:keys` and it fails.

In YAML make your keys look like this:
```yml
:start: Hello, filling for coffins!
```
It will load them as keys.